registrations/models.py

```diff
@@ -94,6 +94,21 @@ def fire_created_metric(sender, instance, created, **kwargs):
 
 
 @receiver(post_save, sender=Registration)
+def fire_source_metric(sender, instance, created, **kwargs):
+    from .tasks import fire_metric
+    if created and 'ussd' in instance.source.name:
```

Comment:
please can we user `source.user.username` instead? That's known to be a slug, and should be descriptive too. Then you won't have to have this logic in here.